platform-jobs/samza/qrcode-image-generator/src/main/java/org/ekstep/jobs/samza/service/QRCodeImageGeneratorService.java

```diff
@@ -123,7 +123,12 @@ private QRCodeGenerationRequest getQRCodeGenerationRequest(Map<String, Object> c
 		if(config.containsKey(QRCodeImageGeneratorParams.qrCodeMarginBottom.name())) {
             qrGenRequest.setQrCodeMarginBottom((Integer) config.get(QRCodeImageGeneratorParams.qrCodeMarginBottom.name()));
         } else {
-            qrGenRequest.setQrCodeMarginBottom(0);
+            qrGenRequest.setQrCodeMarginBottom(appConfig.getInt("qr_image_margin_bottom"));
+        }
+        if(config.containsKey(QRCodeImageGeneratorParams.imageMargin.name())) {
+            qrGenRequest.setImageMargin((Integer) config.get(QRCodeImageGeneratorParams.imageMargin.name()));
+        } else {
+            qrGenRequest.setImageMargin(appConfig.getInt("qr_image_margin"));
```

Comment:
Use constants instead of 'qr_image_margin'.

Reply:
Review comment incorporated in code